autogpt_platform/backend/backend/data/block.py

```diff
@@ -351,6 +352,14 @@ def run(self, input_data: BlockSchemaInputType, **kwargs) -> BlockOutput:
         Run the block with the given input data.
         Args:
             input_data: The input data with the structure of input_schema.
+
+        Kwargs: Currently 14/02/2025 these include
```

Comment:
aside: this should be documented in the new_blocks docs every time something is added to this list